sysutils/exa/Portfile

```diff
@@ -40,20 +40,32 @@ build.pre_args-replace   --frozen --offline
 variant git description {Build with git functionality} {}
 
 variant doc description {Build man pages} {
-    depends_build-append port:pandoc
-    post-build {
-        system -W ${worksrcpath} "${prefix}/bin/pandoc --standalone -f markdown -t man man/exa.1.md > man/exa.1"
-        system -W ${worksrcpath} "${prefix}/bin/pandoc --standalone -f markdown -t man man/exa_colors.5.md > man/exa_colors.5"
+    if {${os.platform} eq "darwin" && ${os.major} < 17 || \
+    ${configure.build_arch} eq "arm64"} {
+        depends_build-append port:go-md2man
+        post-build {
+            system -W ${worksrcpath}/man "${prefix}/bin/go-md2man \
+                -in=${name}.1.md -out=${name}.1"
+            system -W ${worksrcpath}/man "${prefix}/bin/go-md2man \
+                -in=${name}_colors.5.md -out=${name}_colors.5"
+        }
+        notes "
+            The man pages for macOS 10.12 and below or Apple Silicon on may \
+            have minor formatting errors as they were built with go-md2man \
+            instead of pandoc.
```

Comment:
This is good, I like these notes. Just a minor nitpicky typo with _"Apple Silicon on may"_ and then it LGTM.

Reply:
Thanks, that's better? I am not a native English speaker and therefore have some difficulties.

Reply:
That's perfect. Thanks.